core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -1,3 +1,7 @@
+registry.ci.openshift.org/origin/4.11:agent-installer-api quay.io/openshift/origin-agent-installer-api:4.11 quay.io/openshift/origin-agent-installer-api:4.11.0 quay.io/openshift/origin-agent-installer-api:latest
+registry.ci.openshift.org/origin/4.11:agent-installer-agent quay.io/openshift/origin-agent-installer-agent:4.11 quay.io/openshift/origin-agent-installer-agent:4.11.0 quay.io/openshift/origin-agent-installer-agent:latest
```

Comment:
agent-installer-node-agent?

(just to make it feel a little less redundant/stuttery?)

Reply:
That's fine